<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1672</width>
    <height>1182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app6</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QListWidget" name="varsList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="flow">
       <enum>QListView::LeftToRight</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="console">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>333</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="overwriteMode">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="gl_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>600</height>
       </size>
      </property>
      <widget class="QListView" name="listView">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>160</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QToolButton" name="moveButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>MOVE</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="selButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>SEL</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="pickButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PICK</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="gameButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>GAME</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QPushButton" name="xB">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="yB">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="zB">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Z</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="camBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gl_widget2" native="true">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>30</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="savelButton">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reloadShadersBut">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Reload Shaders</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="6">
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Обьект</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>690</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="inspectorTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>641</width>
          <height>401</height>
         </rect>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Новая строка</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Новая строка</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>val</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>type</string>
         </property>
        </column>
       </widget>
       <widget class="QComboBox" name="objectBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>-2</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="setVars">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>730</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Set VARS</string>
        </property>
       </widget>
       <widget class="QLabel" name="classLabel">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>10</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="addAttributes">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>3</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>ДОП АТТРИБУТЫ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="filter">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>3</y>
          <width>71</width>
          <height>17</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>ФИЛЬТР</string>
        </property>
       </widget>
       <widget class="QListWidget" name="objectList">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>30</y>
          <width>631</width>
          <height>71</height>
         </rect>
        </property>
        <property name="flow">
         <enum>QListView::LeftToRight</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Новый элемент</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>730</y>
          <width>261</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>screenshots/1.bmp</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="bHide">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>3</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Hide Unselected</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>101</y>
          <width>503</width>
          <height>19</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="bGroups">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bGroups</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bVeh">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bVeh</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bHelp">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bHelp</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bHidden">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bHidden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bCurves">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bCurves</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bPrim">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bPrim</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bUngroop">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bUngroop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bGeom">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bGeom</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QToolBox" name="dialogBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>540</y>
          <width>641</width>
          <height>431</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>641</width>
           <height>404</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Page 1</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QLineEdit" name="objFilter">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>4</y>
          <width>110</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>car</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Флаги</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>105</width>
          <height>180</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="bColorRendering">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bColorRendering</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bdisable_timer2">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bdisable_timer2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bWire">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bWire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bShading">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bShading</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bdisable_timer1">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bdisable_timer1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bChangedRenderMode">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>bRenderMode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="bNeedUpdVars">
           <property name="text">
            <string>bNeedUpdVars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="gridCB">
           <property name="text">
            <string>Grid</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Симулятор</string>
       </attribute>
       <widget class="QToolButton" name="resetSim">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset Sim</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Quaternion anim</string>
        </property>
       </widget>
       <widget class="QComboBox" name="simMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Полет по законам</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Полет по путевым точкам</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Полет джостиком</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>4</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Управление симулятором полета</string>
        </property>
       </widget>
       <widget class="QToolButton" name="simButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Sim</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="varsEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>471</width>
          <height>381</height>
         </rect>
        </property>
        <property name="plainText">
         <string>--таблица переменных
dt=0.1
p0=7,4,2
angles0=0,0,20
Polet { states=HorizFly,Climb,TurnLeft     times=5,5,5 }
FlyState0{  H=23 v= 10.0,0.0,0.0 }
FlyState1 { Hup = 20  vy = 20 ay = 1 }
FlyState2 { R = 100  A = 20,40,0 }
waypoint0{ ti=0   H=10  We=1 psi0=23 }
waypoint1{ ti=600 H=20  We=2 psi0=24 }
waypoint2{ ti=900 H=30  We=3 psi0=25 }
waypoint3{ ti=1200 H=40 We=5 psi0=26 }
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Законы изменения параметров траектории</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Графики</string>
       </attribute>
       <widget class="QFrame" name="plotFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>560</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1672</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>changeFlag()</slot>
 </slots>
</ui>
